internal/cephfs/nodeserver.go

```diff
@@ -127,13 +128,47 @@ func maybeUnlockFileEncryption(
 	stagingTargetPath string,
 	volID fsutil.VolumeID,
 ) error {
-	if volOptions.IsEncrypted() {
-		log.DebugLog(ctx, "cephfs: unlocking fscrypt on volume %q path %s", volID, stagingTargetPath)
 
+	if volOptions.IsEncrypted() == false {
+		return nil
+	}
+
+	// Define Mutex Lock variables
+	lockName := string(volID) + "-mutexLock"
+	lockCookie := lockName + "-coockie"
+	lockDesc := "Lock for " + string(volID)
+	lockDuration := 40 * time.Second
```

Comment:
should we have some higher value for this one like (150S) default is 180S in kubelet? and make it configurable so user can adjust this one based on the network connectivity with ceph cluster?

Reply:
Should we add flag to make it configurable?

Reply:
My preference is to have a high timeout (150 seconds like it is now should be plenty). Adding a configuration option if really needed can be done later. But hopefully this is sufficient, I expect users that need to have a higher timeout to have more severe problems than failures to setup encryption.

Reply:
works for me as well